### DISCUSSION TITLE
Report bug

### DISCUSSION THREAD

**Discussion by sepehrsdq** [ValveSoftware/Dota2-Gameplay#10297] [General]
Hi when i open dota2 profile for player in game my game is crashing and its a bout all player profiler not just for me
So I can't see my previous games and my mmr, what's the problem? I reinstalled the game, but the problem was not solved

**Comment by Pandfuldell1949** [comment]
Hello , I read your Question so here i put some key points you should try it is help you out ,
Experiencing crashes when accessing player profiles in Dota 2 can be frustrating, but there are several potential factors that could contribute to this issue. Here are some steps you can take to troubleshoot and possibly resolve the problem:

Update Graphics Drivers: Outdated or incompatible graphics drivers can cause crashes. Make sure you have the latest graphics drivers installed for your graphics card.

Verify Game Files: Sometimes, game files can become corrupted. Verify the integrity of your Dota 2 game files through the Steam client to ensure they are not causing the crashes.

Check System Requirements: Ensure that your computer meets the minimum system requirements for Dota 2. If your hardware falls below the recommended specifications, it could lead to crashes.

Temporarily Disable Mods: If you're using mods or custom game content, try disabling them temporarily to see if they are causing the crashes. Sometimes, certain mods can conflict with the game and lead to instability.

Adjust Graphics Settings: Lowering your graphics settings might help reduce the load on your system and prevent crashes. Experiment with different graphics settings to find a stable configuration. [MyJohnDeere Login](https://www.myjdfaccount.net/)

Disable Overlays: Overlays from third-party applications (e.g., Discord, Nvidia GeForce Experience) can sometimes interfere with games. Try disabling any overlays to see if it resolves the issue.

Check for Background Applications: Certain background applications can conflict with games and cause crashes. Close unnecessary applications while playing Dota 2 to see if it helps.

Submit a Bug Report: If the issue persists, consider submitting a bug report to the Dota 2 development team through the official channels. They might be aware of the issue and working on a fix.

Monitor CPU and GPU Temperatures: Overheating can lead to crashes. Monitor your CPU and GPU temperatures using software tools and make sure they are within safe limits.

Contact Support: If none of the above steps work, consider reaching out to Dota 2's official support for further assistance. They may be able to provide specific guidance tailored to your situation.

Remember to document any error messages or patterns you encounter during the crashes, as this information can be helpful for troubleshooting. Additionally, keep an eye on official Dota 2 forums or community discussions to see if others are experiencing similar issues and have found solutions.


I hope you like my answer Thank you.
